ios/RNDemo/NativeMobileSDKBridge.m

```diff
@@ -201,8 +228,6 @@ -(void)startAudioVideo
    if (started && error == nil)
    {
      [logger infoWithMsg:@"RN meeting session was started successfully"];
-
-     [meetingSession.audioVideo startRemoteVideo];
```

Comment:
If you remove this android stream not working on iOS devices.

Reply:
and i also dont understand why he removed this. if we not removing it, will be there unexpected behaviour?